contracts/extensions/MultisigPermissions.sol

```diff
@@ -314,10 +315,25 @@ contract MultisigPermissions is ColonyExtensionMeta, ColonyDataTypes, GetActionS
     return domainSkillThreshold[_domainSkillId];
   }
 
+  function getMotionVoteThreshold(uint256 _motionId, uint8 _role) public view returns (uint256) {
+    Motion storage motion = motions[_motionId];
+    require(bytes32(1 << _role) & motion.requiredPermissions > 0, "multisig-role-not-required");
+    uint256 relevantDomainSkillId = motion.domainSkillId;
+
+    if (getRoleSettingSigs().includes(motion.sig)) {
+      ColonyNetworkDataTypes.Skill memory s = colonyNetwork.getSkill(motion.domainSkillId);
+      if (s.parents.length > 0) {
```

Comment:
Will root architects be able to set root permissions according to this logic?

Reply:
No, but I'm not really sure where you're coming from with this question - can you elaborate, and explain why you think this addition might allow that?

I've added a couple of tests around this, but as ever with anything to do with permissions, keen to know where you're coming from, so if a call will make that easier, let me know.

Reply:
- If memory serves, we don't allow architects to assign permissions in the root domain.

- In this function, it looks as though we are testing to see if the function is a role-setting function, and if so we check the permissions in the parent domain -- unless the domain is root, in which case we check permissions in root.

- If a user had architecture permission in root, would this function allow that permission to vote on a root-level permission change? That is my question.